hieradata/node/cache-2.router.staging.publishing.service.gov.uk.yaml

```diff
@@ -0,0 +1,6 @@
+---
+
+router::gor::add_hosts: false
+router::gor::replay_targets:
+  'www-origin.blue.integration.govuk.digital':
+    ip: 'www-origin.blue.integration.govuk.digital'
```

Comment:
I wonder since we're only using the key we could just set it to:
```
router::gor::replay_targets:
  'www-origin.blue.integration.govuk.digital': {}
```

Reply:
you are right, updating